django_lightweight_queue/management/commands/queue_runner.py

```diff
@@ -22,13 +22,35 @@ class Command(NoArgsCommand):
             help="Total number of machines running the queues"),
         optparse.make_option('--only-queue', action='store', default=None,
             help="Only run the given queue, useful for local debugging"),
+        optparse.make_option('--num-queue-workers', action='store', default=None,
+            help="The number of queue workers to run, only valid when running a specific queue"),
     )
 
     def handle_noargs(self, **options):
         # Django < 1.8.3 leaves options['verbosity'] as a string so we cast to
         # ensure an int.
         verbosity = int(options['verbosity'])
 
+        if (
+            options['num_queue_workers'] is not None and
+            options['only_queue'] is None
+        ):
+            raise CommandError(
+                "A value for 'queue-workers' without a value for 'only-queue' "
+                "has no meaning.",
+            )
+
+        try:
+            only_queue_workers = int(options['num_queue_workers'])
```

Comment:
Could we change `only_queue_workers` throughout to have `num` in it somewhere? E.g. `only_queue_num_workers`